src/core/thread/data_poll_manager.cpp

```diff
@@ -201,6 +201,8 @@ void DataPollManager::HandlePollSent(ThreadError aError)
             shouldRecalculatePollPeriod = true;
         }
 
+        otLogInfoMac(GetInstance(), "Sent data poll");
```

Comment:
`DataPollManager::SendDataPoll()` also has a `Sent poll` log message.  Should consider whether to keep both. 
 If we want to keep both, we should make their difference clear and also reconcile whether to keep the log level the same or different.